### DISCUSSION TITLE
[Support]: Fedora 40 Container-Deployment: s6-linux-init-shutdown: fatal: unable to talk to shutdownd: Operation not permitted

### DISCUSSION THREAD

**Discussion by Marcwa19197** [blakeblackshear/frigate#11113] [General Support] [answered]
### Describe the problem you are having

Hello,

since updating to Fedora 40 from Fedora 39 my Container is crashing after a few hours of working smooth. After a restart of the container, it is running well for a few hours.

Latest Loglines:
```
s6-linux-init-shutdown: fatal: unable to talk to shutdownd: Operation not permitted
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
```

What does this Connection-Refuses mean?

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
Not related, no change done.
```

### Relevant log output

```shell
s6-linux-init-shutdown: fatal: unable to talk to shutdownd: Operation not permitted
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
s6-fdholder-retrieve: fatal: unable to connect to a fd-holder daemon at ../s6rc-fdholder/s: Connection refused
```

### FFprobe output from your camera

```shell
Not related, no change done.
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Reolinks / Not related

### Any other information that may be helpful

SeLinux is enabled, but not showing any Audits. Was configured well before updating to Fedora 40.

**Comment by skDn** [comment]
Any solution to this? I'm seeing the same logs. The frigate container is crashing after some time. I'm running 0.14.1 on Debian 12, rootless docker with docker compose. I'm also seeing a bunch of 400 requests, not sure if it is some sort of a health check:
`2024-12-28 01:23:53.407138885  127.0.0.1 - - [28/Dec/2024:01:23:53 +0000] "" 400 0 "-" "-" "-"`

**Comment by skDn** [comment, marked as answer]
If anybody ends up here, I managed to figure out my issue and it had nothing to do with Frigate. It was my docker installation. Docker was stopping all containers when I was terminating my SSH session. If you are running rootless docker, just have to execute `sudo loginctl enable-linger <user>`
